### ISSUE TITLE
Perform simple accessibility review

### ISSUE THREAD

**Issue by FredrikAugust** [IT2901-SINTEF01/frontend#63]
### Description
Even though it's not crucial for our application, we should strive to hit at least the most basic accessibility goals.

The task is to perform a review of the accessibility (or lack thereof) and provide a report to the frontend developers through an issue with subtasks. Might also be helpful to include a PDF or similar.

### Subtasks
- [x] Perform review
- [x] Create report
- [x] Create issue
- [x] Document findings if you see fit

**Comment by blauks** [comment]
@thomran how is this going?

**Comment by thomran** [comment]
Will begin today and aim to finish the review within Wednesday. Will that be ok?

**Comment by blauks** [comment]
Sounds good to me
